deploy/README.md

```diff
@@ -85,6 +85,11 @@ To install infrastructure execute install.sh script as follows
 
 ```
 
+You will be prompted to enter in a password for postgres. If left empty, an auto generated password will be created and stored in the keyvault. The password requirements are listed below:
+- Between 8 characters and 128 characters
+- Must contain characters from three of the following categories – English uppercase letters, English lowercase letters, numbers (0-9), and non-alphanumeric characters (!, $, #, %, etc.). 
+- Your password cannot contain all or part of the login name. Part of a login name is defined as three or more consecutive alphanumeric characters.
+
```

Comment:
Can we add a brief message to say what is the benefit of integrating Postgres? Also, can it be optional?

Reply:
Maybe, we'll have to remove the below line from readme.md and also in any other places like reference architecture 
```
Note: Currently, this deployment does not deploy Azure Database for PostgreSQL for post-analysis.
```

Reply:
Maybe, we'll have to remove the below line from readme.md and also in any other places like reference architecture 
```
Note: Currently, this deployment does not deploy Azure Database for PostgreSQL for post-analysis.
```